draft-ietf-quic-transport.md

```diff
@@ -3061,14 +3061,13 @@ of doing so.
 
 Packets containing only ACK frames are not congestion controlled, so there are
 limits on how frequently they can be sent.  An endpoint MUST NOT send more than
-one ACK-frame-only packet in response to receiving an ack-eliciting packet (one
-containing frames other than ACK and/or PADDING).  An endpoint MUST NOT send a
-packet containing only an ACK frame in response to a non-ack-eliciting packet
-(one containing only ACK and/or PADDING frames), even if there are packet gaps
-which precede the received packet. Limiting ACK frames avoids an infinite
-feedback loop of acknowledgements, which could prevent the connection from ever
-becoming idle. However, the endpoint acknowledges non-ack-eliciting packets when
-it sends an ACK frame.
+one ACK-frame-only packet in response to receiving an ack-eliciting packet.  An
+endpoint MUST NOT send a non-ack-eliciting packet in response to a non-ack-
+eliciting packet, even if there are packet gaps which precede the received
```

Comment:
A nit:  Though this hyphenation looks sane when presented this way, it won't be guaranteed to render nicely.  It might appear as `non-ack- eliciting` (note the extra space).  I think that you should remove the wrapping on the hyphen.